### PR TITLE
refactor: proper extends rootClasses

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context
- refactoring smol stuff

### <samp>🤖 Generated by Copilot at f68c27c</samp>

Refactored the `NeoField` component to use JavaScript instead of TypeScript and improved its styling and functionality with Oruga classes. Renamed the file `NeoInput.vue` to `NeoField.vue` to match the component name.

### <samp>🤖 Generated by Copilot at f68c27c</samp>

> _`NeoField` transforms_
> _From TypeScript to JavaScript_
> _Oruga in autumn_

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]

**Comment by roiLeo** [comment]
made some changes in #6160 could you check if UI breaks?

File: libs/ui/src/components/NeoInput/NeoField.vue [hunk 1]
```diff
@@ -1,11 +1,29 @@
-<script lang="ts">
+<script>
```
Comment (daiagi) [inline libs/ui/src/components/NeoInput/NeoField.vue:1]: why not 'ts'?

**Review by daiagi — APPROVED** [review]

**Comment by preschian** [comment]
> made some changes in #6160 could you check if UI breaks?

fixed "expanded" props. the rest, looks same

image

**Comment by roiLeo** [comment]
Lets go!

**Pull request merged**